app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("app/cinema.sqlite")
```

Comment:
The database connection is opened in the constructor, but there is no method to close it. Consider implementing a `close` method to close the connection when it's no longer needed to prevent resource leaks.